bundles/org.eclipse.equinox.slf4j/src/org/eclipse/equinox/slf4j/EquinoxLogger.java

```diff
@@ -32,213 +34,77 @@ public boolean isTraceEnabled() {
-	public void debug(String format, Object... argArray) {
-		if (isDebugEnabled()) {
-			FormattingTuple tp = MessageFormatter.arrayFormat(format, argArray);
-			logger.debug(tp.getMessage(), tp.getThrowable());
-		}
-	}
-
 	@Override
 	public boolean isInfoEnabled() {
 		return logger != null && logger.isInfoEnabled();
 	}
 
-	@Override
-	public void info(String msg) {
-		if (isInfoEnabled()) {
-			logger.info(msg);
-		}
-	}
-
-	@Override
-	public void info(String msg, Throwable t) {
-		if (isInfoEnabled()) {
-			logger.info(msg, t);
-		}
-	}
-
-	@Override
-	public void info(String format, Object arg1) {
-		if (isInfoEnabled()) {
-			FormattingTuple tp = MessageFormatter.format(format, arg1);
-			logger.info(tp.getMessage(), tp.getThrowable());
-		}
-	}
-
-	@Override
-	public void info(String format, Object arg1, Object arg2) {
-		if (isInfoEnabled()) {
-			FormattingTuple tp = MessageFormatter.format(format, arg1, arg2);
-			logger.info(tp.getMessage(), tp.getThrowable());
-		}
-	}
-
-	@Override
-	public void info(String format, Object... argArray) {
-		if (isInfoEnabled()) {
-			FormattingTuple tp = MessageFormatter.arrayFormat(format, argArray);
-			logger.info(tp.getMessage(), tp.getThrowable());
-		}
-	}
-
 	@Override
 	public boolean isWarnEnabled() {
 		return logger != null && logger.isWarnEnabled();
 	}
 
-	@Override
-	public void warn(String msg) {
-		if (isWarnEnabled()) {
-			logger.warn(msg);
-		}
+	public boolean isErrorEnabled() {
+		return logger != null && logger.isErrorEnabled();
 	}
 
-	@Override
-	public void warn(String msg, Throwable t) {
-		if (isWarnEnabled()) {
-			logger.warn(msg, t);
-		}
+	public boolean isTraceEnabled(Marker marker) {
+		return isTraceEnabled();
 	}
 
 	@Override
-	public void warn(String format, Object arg1) {
-		if (isWarnEnabled()) {
-			FormattingTuple tp = MessageFormatter.format(format, arg1);
-			logger.warn(tp.getMessage(), tp.getThrowable());
-		}
+	public boolean isDebugEnabled(Marker marker) {
+		return isDebugEnabled();
 	}
 
 	@Override
-	public void warn(String format, Object arg1, Object arg2) {
-		if (isWarnEnabled()) {
-			FormattingTuple tp = MessageFormatter.format(format, arg1, arg2);
-			logger.warn(tp.getMessage(), tp.getThrowable());
-		}
+	public boolean isInfoEnabled(Marker marker) {
+		return isInfoEnabled();
 	}
 
 	@Override
-	public void warn(String format, Object... argArray) {
-		if (isWarnEnabled()) {
-			FormattingTuple tp = MessageFormatter.arrayFormat(format, argArray);
-			logger.warn(tp.getMessage(), tp.getThrowable());
-		}
+	public boolean isWarnEnabled(Marker marker) {
+		return isWarnEnabled();
 	}
 
 	@Override
-	public boolean isErrorEnabled() {
-		return logger != null && logger.isErrorEnabled();
+	public boolean isErrorEnabled(Marker marker) {
+		return isErrorEnabled();
 	}
 
 	@Override
-	public void error(String msg) {
-		if (isErrorEnabled()) {
-			logger.error(msg);
-		}
+	protected String getFullyQualifiedCallerName() {
+		return this.name;
```

Comment:
The [SimpleLogger](https://github.com/qos-ch/slf4j/blob/183aaa507040ce6c61f70762c13e7d11aa4fd54e/slf4j-simple/src/main/java/org/slf4j/simple/SimpleLogger.java#L471-L473) only return `null` here, as there is no javadoc what this method is used for one might either check other Slf4J implementations or also just return null instead of the logger name.